stories/ConfigureRelatedItems.stories.tsx

```diff
@@ -0,0 +1,115 @@
+/* eslint react/prop-types: 0 */
```

Comment:
we can disable this on .tsx in the stories globally maybe?

Reply:
Will do.